RELEASE-NOTES.txt

```diff
@@ -16,6 +16,7 @@
 * [*] Fix notice overlapping the ActionSheet that displays the More Actions in the Editor. [#16658]
 * [*] The quick action buttons will be hidden when iOS is using a accessibility font sizes. [#16701]
 * [*] Block Editor: Improve unsupported block message for reusable block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3621]
+* [**] Fix incorrect block insertion point after blurring the post title field. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3640]
```

Comment:
I see the rest of the entry items related to Gutenberg have the prefix Block Editor:, I'm not sure if it's a convention but we might want to include it, wdyt

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/16723/commits/e31f42df3ed55b3698c4397800bc57e914041e75.